Supporting non-python CDK projects (#10)
Sceptre/sceptre-cdk-handler: README.md

The template "type" for this handler is `cdk`.

### Python Stack Classes vs. cdk.json

> The template `path` argument can either direct to a Python file with a `SceptreCdkStack class` (see
> "Making your Python stack class" below) or it can point a `cdk.json` file. Template synthesis and

Reviewer: ```suggestion
The template `path` argument can either specify a Python file with a `SceptreCdkStack class` (see
"Making your Python stack class" below) or it can specify a `cdk.json` file. Template synthesis and
```

    # and then you can add your resources...
```

> ### Importing from other modules

Author: This section I just moved up from lower in the Readme. It seemed more appropriately placed after the section on creating your stack class.

    - "image_asset_tag_prefix"
    - "template_bucket_name"

### Passing Data to a CDK Stack

> There are three methods for passing data to a CDK Stack; using `sceptre_user_data` or CloudFormation parameters:

Reviewer: ```suggestion
There are three methods for passing data to a CDK Stack; using `sceptre_user_data`, CDK Context or CloudFormation parameters:
```

Author: Ha. Apparently I can't count.

#### Sceptre User Data

Data can be passed to a CDK stack using the `sceptre_user_data` block of the Sceptre stack config.
This will be resolved when the template is synthesized and can contain complex objects. Since
`sceptre_user_data` is a resolvable property, you can use resolvers to pass values from other

> deployed stacks and other sources as well.x`

Reviewer: ```suggestion
deployed stacks and other sources as well.`
```

> **Note**: `sceptre_user_data` is not supported if your `path` is to a `cdk.json` file.

Reviewer: ```suggestion
**Note**: `sceptre_user_data` is not supported if your `path` points to a `cdk.json` file.
```

#### CDK Context

Data can be passed to a CDK application (and accessed in your stack) via the `context` template
handler argument. Since Template handler arguments are resolvable, you can also use resolvers here.

**Note**: Complex context values (in lists and dicts) are only supported for Python stacks. If your

> `path` is to a `cdk.json` file, only string values are supported.

Reviewer: ```suggestion
`path` points to a `cdk.json` file, only string values are supported.
```

you name your stack according to Sceptre's naming conventions, `validate`, `diff`, and more! Basically,
the CDK stack can be managed using Sceptre just like any other.

> ### Importing from other files

Author: This was just moved higher, not actually removed.
